skimage/io/tests/test_pil.py

```diff
@@ -165,9 +167,14 @@ def test_jpg_quality_arg():
         assert sim > 0.99
 
 
+@pytest.mark.xfail(
+    Version(np.__version__) >= Version('2.0.0.dev0'),
+    reason='use_plugin("pil") seems broken for now and tifffile is used instead! '
+    'tifffile uses deprecated attribute `ndarray.newbyteorder`',
+)
 def test_imread_uint16_big_endian():
```

Comment:
Despite this being the test for the PIL plugin, for some reason this triggers an error in tifffile which leads me to believe that the `use_plugin` function may be broken... but that's something for another issue and PR.

Reply:
See https://github.com/scikit-image/scikit-image/issues/7305.